### ISSUE TITLE
Can I move multiple rows to another table at the same time?

### ISSUE THREAD

**Issue by alxbbarbosa** [olifolkerd/tabulator#1259]
Hi oli

How can I move multiple rows between different tables at once? I need to provide a way for users to select one or more records and then move to another table. For example, when users need to create a new service request, they will need to select service details in a first table, and then populate another table with the selected details. About moving rows between different tables, I get it. But I could not define how to move two or more rows between the tables at the same time. Is it possible to do that?

In other words, what I want to do is a mix using "selectable rows" and "moving rows between tables". But it is not working as I would expect

![image](https://user-images.githubusercontent.com/16040564/42526345-ec6b1146-844b-11e8-9b55-deaeaba1f2ff.png)

Is this the way or is there another correct way to do this?

**Comment by olifolkerd** [comment]
Hey @alxbbarbosa 

Thanks for getting in touch.

At the moment i am afraid that isnt possible, but that functionality will be coming in the next release.

Cheers

Oli :)
